src/components/Personalization/handlers/createDomActionHandler.js

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import { DEFAULT_CONTENT_ITEM, DOM_ACTION } from "../constants/schema";
+
+export default ({
+  next,
+  modules,
+  storeClickMetrics,
+  preprocess
+}) => proposition => {
+  const { items = [] } = proposition.getHandle();
+
+  items.forEach((item, index) => {
+    const { schema, data } = item;
+    if (schema === DEFAULT_CONTENT_ITEM) {
+      proposition.includeInDisplayNotification();
+      proposition.addRenderer(index, () => undefined);
+    }
+    const { type, selector } = data || {};
```

Comment:
For a use-case of a form-based composed HTML offer for global mbox we receive a proposition item that looks like this:
```
{
    "type": "setHtml",
    "format": "application/vnd.adobe.target.dom-action",
    "content": "\"Hello world\"",
    "selector": "head",
    "prehidingSelector": "head"
}
```
When this is rendered, we get a blank HTML `head` tag which results in a broken page since all the styles and scripts are removed.
As far as I can tell previously we had a logic that transformed for `head `selector the `setHtml` action type into `appendHtml`, to avoid messing up the HTML `head`.